src/wp-content/themes/twentyseventeen/functions.php

```diff
@@ -451,7 +455,8 @@ function twentyseventeen_colors_css_wrap() {
  */
 function twentyseventeen_scripts() {
 	// Add custom fonts, used in the main stylesheet.
-	wp_enqueue_style( 'twentyseventeen-fonts', twentyseventeen_fonts_url(), array(), null );
+	$font_version = ( 0 === strpos( (string) twentyseventeen_fonts_url(), get_template_directory_uri() . '/' ) ) ? '20230328' : null;
```

Comment:
Can we use `str_starts_with` here?

Reply:
Unfortunately, no. These themes should work with older versions of both PHP and WordPress.